### PR TITLE
fix sup/sub markdown bug

### PR DESCRIPTION
**Pull request by yCobanoglu**

### CODE REVIEW

**Comment by yCobanoglu** [comment]
[Error: Renderer for type `sup` not defined or is not renderable](https://trello.com/c/x0s0bkTo/516-error-renderer-for-type-sup-not-defined-or-is-not-renderable)

File: src/common/InlineMarkdown.tsx [hunk 1]
```diff
@@ -13,6 +13,11 @@ export default function InlineMarkdown({
       plugins={[remarkSubSuper]}
       disallowedTypes={['paragraph']}
       unwrapDisallowed
+      renderers={{
+        paragraph: 'span',
```
Comment (jlaasonen) [inline src/common/InlineMarkdown.tsx:17]: This seems redundant due `disallowedTypes` and `unwrapDisallowed`.

Comment (jlaasonen) [reply src/common/InlineMarkdown.tsx:17]: @yCobanoglu how was this resolved? There are no changes and no comment explaining why it has to be so.

**Review by jlaasonen — CHANGES_REQUESTED** [review]
See comment.

File: src/common/InlineMarkdown.tsx (hunk 1 above)
Comment (yCobanoglu) [reply src/common/InlineMarkdown.tsx]: ok now i removed paragraph span

**Review by jlaasonen — APPROVED** [review]

**Pull request merged**